migration_original/ODS1Stage/tables/Show/SOLRProvider_Freeze/spu_translated_SOLRProvider_Freeze.sql

```diff
@@ -0,0 +1,411 @@
+                    HasProviderSpecialtyFacility5StarXML,
+                    DisplayPatientExperienceSurveyOverallScore,
+                    ProductGroupCode,
+                    SponsorCode,
+                    FacilityCode,
+                    SearchSponsorshipXML,
+                    ProductCode,
+                    VideoXML,
+                    OASXML,
+                    SuppressSurvey,
+                    ProviderURL,
+                    ImageXML,
+                    AdXML,
+                    HasProfessionalOrganizationXML,
+                    ProfessionalOrganizationXML,
+                    ProviderProfileViewOneYear,
+                    PracticingSpecialtyXML,
+                    CertificationXML,
+                    HasPracticingSpecialtyXML,
+                    HasCertificationXML,
+                    PatientExperienceSurveyOverallStarValue,
+                    ProviderBiography,
+                    DisplayStatusCode,
+                    HealthInsuranceXML_v2,
+                    ProviderDEAXML,
+                    ProviderTypeXML,
+                    SubStatusCode,
+                    DuplicateProviderCode,
+                    DeactivationReason,
+                    ProcedureHierarchyXML,
+                    ConditionHierarchyXML,
+                    ProcMappedXML,
+                    CondMappedXML,
+                    PracSpecHeirXML,
+                    AboutMeXML,
+                    HasAboutMeXML,
+                    PatientVolume,
+                    HasMalpracticeState,
+                    ProcedureCount,
+                    ConditionCount,
+                    AvailabilityXML,
+                    VideoXML2,
+                    AvailabilityStatement,
+                    IsInClientMarket,
+                    HasOAR,
+                    IsMMPUser,
+                    NatlAdvertisingXML,
+                    APIXML,
+                    DIHGroupNumber,
+                    SubStatusDescription,
+                    DEAXML,
+                    EmailAddressXML,
+                    DegreeXML,
+                    HasSurveyXML,
+                    HasDEAXML,
+                    HasEmailAddressXML,
+                    ClientCertificationXML,
+                    HasGoogleOAS,
+                    HasVideoXML2,
+                    HasAboutMe,
+                    ConversionPathXML,
+                    SearchBoostSatisfaction,
+                    SearchBoostAccessibility,
+                    IsPCPCalculated,
+                    FAFBoostSatisfaction,
+                    FAFBoostSancMalp,
+                    FFDisplaySpecialty,
+                    FFPESBoost,
+                    FFMalMultiHQ,
+                    FFMalMulti,
+                    CLinicalFocusXML,
+                    ClinicalFocusDCPXML,
+                    SyndicationXML,
+                    TeleHealthXML ';
+
+select_statement := 'WITH CTE_ClientCode AS (SELECT
```

Comment:
Select in a new line
```suggestion
select_statement := 'WITH CTE_ClientCode AS (
                     SELECT
```

Reply:
Done